android/java/org/chromium/chrome/browser/crypto_wallet/fragments/PortfolioFragment.java

```diff
@@ -378,49 +369,49 @@ private void updatePortfolioGraph() {
 
     private void updatePortfolioGetPendingTx(boolean getPendingTx) {
         KeyringService keyringService = getKeyringService();
-        JsonRpcService jsonRpcService = getJsonRpcService();
-        assert keyringService != null && jsonRpcService != null;
-
-        keyringService.getKeyringInfo(BraveWalletConstants.DEFAULT_KEYRING_ID, keyringInfo -> {
-            AccountInfo[] accountInfosTemp = new AccountInfo[] {};
-            if (keyringInfo != null) {
-                accountInfosTemp = keyringInfo.accountInfos;
-            }
-            final AccountInfo[] accountInfos = accountInfosTemp;
-            Activity activity = getActivity();
-            if (!(activity instanceof BraveWalletActivity)) return;
-            if (mPortfolioHelper == null) {
-                mPortfolioHelper =
-                        new PortfolioHelper((BraveWalletActivity) activity, accountInfos);
-            }
+        assert keyringService != null;
 
-            jsonRpcService.getNetwork(CoinType.ETH, selectedNetwork -> {
-                mPortfolioHelper.setSelectedNetwork(selectedNetwork);
-                mPortfolioHelper.calculateBalances(() -> {
-                    final String fiatSumString = String.format(
-                            Locale.getDefault(), "$%,.2f", mPortfolioHelper.getTotalFiatSum());
-                    PostTask.runOrPostTask(UiThreadTaskTraits.DEFAULT, () -> {
-                        mFiatSumString = fiatSumString;
-                        mBalance.setText(mFiatSumString);
-                        mBalance.invalidate();
-
-                        setUpCoinList(mPortfolioHelper.getUserAssets(),
-                                mPortfolioHelper.getPerTokenCryptoSum(),
-                                mPortfolioHelper.getPerTokenFiatSum());
-
-                        updatePortfolioGraph();
+        final NetworkInfo selectedNetwork =
+                mWalletModel.getCryptoModel().getNetworkModel().mDefaultNetwork.getValue();
+        if (selectedNetwork == null) {
+            return;
+        }
+        keyringService.getKeyringInfo(
+                Utils.getKeyringForCoinType(selectedNetwork.coin), keyringInfo -> {
```

Comment:
nit: use [WalletUtils](https://github.com/brave/brave-core/blob/062a8d984f6d38ca9c399994eea1f452edff6d0d/android/java/org/chromium/chrome/browser/crypto_wallet/util/WalletUtils.java) for methods specific to wallet, utils is completely bloated with all sorts of helper methods.

Reply:
not sure what is the difference between Utils that were done for the wallet initially and WalletUtils. We need to call it more specific for every type of Utils as it's confusing as Utils is also a bunch of helper functions for wallet. Le't leave that for the upcoming refactor and create diff utils files for diff types.